galaxy/values.yaml

```diff
@@ -267,6 +267,31 @@ extraEnv: []
   # - name: EXAMPLE_ENV
   #   value: MY_VALUE
 
+#- User defined CronJobs
+cronJobs:
+  chown:
+    schedule: "0 * * * *"
+    securityContext:
+      runAsUser: 0
+    command:
+      - /usr/bin/chown
+    args:
+      - "-R"
+      - "galaxy"
+      - "/galaxy/server/database"
+#- Maintenance tasks that should be run periodically to prevent storage from being exhausted
+maintenance:
```

Comment:
Cam we make the maintenance jobs regular cronjobs as well, just with a "known" name? Something like:
```
cronJobs:
  maintenance:
    schedule: "0 * * * *"
    extraSettings:
       days: 7
    securityContext:
      runAsUser: 0
      command:
        - "/bin/bash"
        - "-c"      
      args:
        - "/galaxy/server/scripts/maintenance.sh"
        - "--no-dry-run"
        - "--days"
        - {{ .Values.cronJobs.maintenance.extraSettings.days | quote }}
```

And just tpl the `args` value in the cronjob template?

Reply:
I guess we could.  Should we do the same for the tmp clean up script as well then?  I assume we may as well.

Reply:
Done.